index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav class="navigation">
+            <a class="logo" href="#">Hami</a>
```

Comment:
- [ ] Good job taking `a` tag and putting your logo inside it but kindly take this `a` tag out of the `nav`. This would make your design follow best practices and semantics.